src/common/reply.h

```diff
@@ -18,7 +18,8 @@
     (RPL_META_MESSAGE(servername, "004", nick + " " + servername + " " + version + " " + user_modes + " " + channel_modes))
 
 #define RPL_NOTOPIC_331(servername, channel) (RPL_META_MESSAGE(servername, "331", channel + " :No topic is set"))
-#define RPL_TOPIC_332(servername, channel, topic) (RPL_META_MESSAGE(servername, "332", channel + " :" + topic))
+#define RPL_TOPIC_332(nick, user, servername, channel, topic) \
+    (RPL_META_MESSAGE(nick + "!" + user + "@" + servername, "332 TOPIC", channel + " " + topic))
```

Comment:
Reference | Content after `:nick!user@serverHost 332 ` in `RPL_TOPIC`
--- | ---
Per IRCv3 | `<client> <channel> :<topic>`
Per RFC2812 | `<channel> :<topic>`

Note: `RPL_NOTOPIC` is following RFC2812

So the current implementation in this PR seems incorrect.

Reply:
change